client/lib/directly/index.js

```diff
@@ -0,0 +1,50 @@
+/**
+ * External dependencies
+ */
+import config from 'config';
+
+/**
+ * Internal dependencies
+ */
+import { initializeDirectly } from './vendor';
+
+const DEFAULT_RTM_WIDGET_OPTIONS = {
+	id: config( 'directly_rtm_widget_id' ),
+	displayAskQuestion: false
+};
+
+const directly = {
+	initialize: function( config = {} ) {
+		initializeDirectly( Object.assign( {}, DEFAULT_RTM_WIDGET_OPTIONS, config ) );
+	},
+
+	askQuestion: function( questionText, name, email ) {
+		if ( ! window.DirectlyRTM ) {
+			return;
+		}
+		window.DirectlyRTM( 'askQuestion', { questionText, name, email } );
```

Comment:
Nope, it's a black box. All you can do is fire commands in which are `postMessage()`d into the iframe. There's no advertised way to hook into any events or callbacks (and a quick glance of source seems like there's no way for us to finagle some hidden way in either).